Makefile

```
@@ -40,14 +41,34 @@ start:
 KEYLIME_CERT_DIR := /var/lib/keylime/cv_ca
 CERT_FILES := cacert.crt client-cert.crt client-private.pem
 
-setup-certs:
-	@echo "Granting read access to Keylime certificates for user '$(USER)'..."
+SYSTEMD_SERVICE := /etc/systemd/system/keylime-mcp-certs.service
+
+setup-certs: setup-certs-session
```

checkmake (0.3.2)

[warning] Target body for "setup-certs" exceeds allowed length of 5 lines (16).

(maxbodylength)